contracts/src/BeefyClient.sol

```diff
@@ -212,9 +233,14 @@ contract BeefyClient {
         external
     {
         ValidatorSet memory vset;
+        uint16 numRequiredSignatures;
```

Comment:
Looks we renamed this incorrectly.
```suggestion
        uint16 signatureUsageCount;
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/956/commits/3da2bef5c6b3b13b02a5f4cd45a63e375c8c39e9